onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -1034,6 +1058,11 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
                 "or present state for self attention value with shape (batch_size, num_heads, total_sequence_length, head_size)",
                 "T",
                 OpSchema::Optional)
+        .Output(3,
```

Comment:
You will need update documents (You can find the updated documents in artifacts of `Windows GPU Doc Gen CI Pipeline` for this PR).